pyscript.core/package.json

```diff
@@ -5,7 +5,7 @@
     "main": "./cjs/index.js",
     "types": "./types/index.d.ts",
     "scripts": {
-        "server": "npx static-handler --cors --coep --coop .",
+        "server": "npx static-handler --cors --coep --coop --corp .",
```

Comment:
Wow nice that it can handle the required headers with such simple cli flags.

Reply:
it's too simple and all flags are ad-hoc for basic use cases too ... I haven't spent too much time in there to be honest, but it could be better (I suppose)